docs/conf.py

```diff
@@ -0,0 +1,323 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
```

Comment:
Wrong copyright header; should not read Invenio, but Invenio-JSONSchemata, provided that we select this term.  (see repository name)